draft-ietf-rtcweb-jsep.xml

```diff
@@ -603,10 +603,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             type.
           </t>
           <t>As an example, consider a system with a HD-capable, multiformat
-          video decoder, where the application has constrained the received
-          track to at most 360p. In this case, the implemention would generate this
-          attribute:</t>
+            video decoder, where the application has constrained the received
+            track to at most 360p. In this case, the implemention would generate this
+            attribute:</t>
           <t>a=imageattr:* recv [x=[16:640],y=[16:360],q=1.0]</t>
+          <t>Also to note, the above example implies that the receiver is capable 
+            of decoding all the image screen resolutions ranging from 16x16 
+            pixels up to 640x360 pixels.</t>
+          <t>In scenarios where the receiver would want to constrain the possible
```

Comment:
The paragraph above mentions how a receiver might want to constrain the possible resolutions, so this sentence feels out of place.